packages/teleport/src/Discover/SelectResource/SelectDBType.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
```

Comment:
Code was moved here so that it can be used for the next step of Discover even though it's technically unused at this point.